src/components/FirstRegistrationForm.tsx

```diff
@@ -0,0 +1,95 @@
+import { Formik, Form, Field } from 'formik';
+import Email from "./Email";
+import PasswordAndConfirm from './PasswordAndConfirm';
+import Checkbox from './Checkbox';
+import './FirstRegistrationForm.scss';
+import { isEmailExists } from '../services/api/isEmailExists';
+
+interface FormValues {
+  email: string;
+  password: string;
+  confirmPassword: string;
+  termsAndConditions: boolean;
+}
+
+interface FormErrors {
+  [key: string]: any
```

Comment:
Why any? It looks to me that all errors are string...

Reply:
fixed